code/game/objects/items/weapons/storage/stickersheets.dm

```diff
@@ -16,6 +16,22 @@
 		/obj/item/sticker
 	)
 
+// List/Procs to list all stickers for custom stickersheet(similar to lunchbox code)
+var/list/stickersheet_stickers_ = typesof(/obj/item/sticker)
+
+/proc/stickersheet_stickers()
+	if(!(stickersheet_stickers_[stickersheet_stickers_[1]]))
+		stickersheet_stickers_ = init_sticketsheet_stickers(stickersheet_stickers_)
+	return stickersheet_stickers_
+
+/proc/init_sticketsheet_stickers(var/list/stickers)
```

Comment:
DMDoc and why is this a global proc instead of being under /datum/gear/toy/stickersheet_custom?